neo/Prompt/Commands/Tokens.py

```diff
@@ -2,10 +2,79 @@
 from neo.Prompt.Utils import get_asset_id, get_from_addr, get_tx_attr_from_args
 from neo.Wallets.NEP5Token import NEP5Token
 from neocore.Fixed8 import Fixed8
+from neocore.UInt160 import UInt160
 from prompt_toolkit import prompt
 from decimal import Decimal
 from neo.Core.TX.TransactionAttribute import TransactionAttribute, TransactionAttributeUsage
 import binascii
+from neo.Prompt.CommandBase import CommandBase, CommandDesc, ParameterDesc
+from neo.Prompt.PromptData import PromptData
+from neo.Prompt.Utils import get_arg
+from neo.Implementations.Wallets.peewee.Models import NEP5Token as ModelNEP5Token
+import peewee
+
+
+class CommandWalletToken(CommandBase):
+    def __init__(self):
+        super().__init__()
+        self.register_sub_command(CommandTokenDelete())
+
+    def command_desc(self):
+        return CommandDesc('token', 'various token operations')
+
+    def execute(self, arguments):
+        item = get_arg(arguments)
+
+        if not item:
+            print(f"Please specify an action. See help for available actions")
+            return
+
+        try:
+            return self.execute_sub_command(item, arguments[1:])
+        except KeyError:
+            print(f"{item} is an invalid parameter")
+            return
+
+
+class CommandTokenDelete(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) != 1:
+            print("Please specify the required parameter")
+            return
```

Comment:
Wouldn't it be better to return `False` in early returns since `DeleteNEP5Token` returns a boolean?